settings.gradle

```diff
@@ -39,6 +39,7 @@ include 'h2o-security'
 include 'h2o-logger'
 include 'h2o-genmodel-extensions:jgrapht'
 include 'h2o-k8s'
+include 'h2o-genmodel-extensions:deepwater'
```

Comment:
this should imho follow the xgboost naming convention, h2o-genmodel-ext-deepwater

Reply:
Right now, it seems like me and @koniecsveta set the conventions :rofl: 

Of course, I'll rename it.